app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -365,6 +325,46 @@ public static Snackbar showThemedSnackbar(
     return snackbar;
   }
 
+  public static Snackbar showCutCopySnackBar(
+      MainActivity mainActivity,
+      CharSequence text,
+      int length,
+      @StringRes int actionTextId,
+      Runnable actionCallback,
+      Runnable cancelCallback) {
+
+    final Snackbar snackbar =
+        Snackbar.make(mainActivity.findViewById(R.id.content_frame), "", length);
+
+    View customSnackView =
+        View.inflate(mainActivity.getApplicationContext(), R.layout.snackbar_view, null);
+    snackbar.getView().setBackgroundColor(Color.TRANSPARENT);
+
+    Snackbar.SnackbarLayout snackBarLayout = (Snackbar.SnackbarLayout) snackbar.getView();
+    snackBarLayout.setPadding(0, 0, 0, 0);
+
+    Button actionButton = (Button) customSnackView.findViewById(R.id.snackBarActionButton);
+    Button cancelButton = (Button) customSnackView.findViewById(R.id.snackBarCancelButton);
+    TextView textView = (TextView) customSnackView.findViewById(R.id.snackBarTextTV);
+
+    actionButton.setText(actionTextId);
+    textView.setText(text);
+
+    actionButton.setOnClickListener(v -> actionCallback.run());
+    cancelButton.setOnClickListener(v -> cancelCallback.run());
+
+    snackBarLayout.addView(customSnackView, 0);
+
+    if (mainActivity.getAppTheme().equals(AppTheme.LIGHT)) {
```

Comment:
A few themes missing here

Reply:
@EmmanuelMess This push contains the black and white one as @VishalNehra asked for checking how it looks...